ui/src/routes/admin/colors/index.tsx

```diff
@@ -123,6 +132,203 @@ const ColorsComponent = () => {
+                      className={`border ${getEcosystemBackgroundColor(p)}`}
+                    >
+                      {p}
+                    </Badge>
+                  ))}
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell>C++</TableCell>
+                <TableCell className='flex gap-2'>
+                  {['Carthage', 'Conan'].map((p) => (
+                    <Badge
+                      className={`border ${getEcosystemBackgroundColor(p)}`}
+                    >
+                      {p}
+                    </Badge>
+                  ))}
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell>Dart</TableCell>
+                <TableCell className='flex gap-2'>
+                  {['Pub'].map((p) => (
+                    <Badge
+                      className={`border ${getEcosystemBackgroundColor(p)}`}
+                    >
+                      {p}
+                    </Badge>
+                  ))}
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell>Go</TableCell>
+                <TableCell className='flex gap-2'>
+                  {['GoMod'].map((p) => (
+                    <Badge
+                      className={`border ${getEcosystemBackgroundColor(p)}`}
+                    >
+                      {p}
+                    </Badge>
+                  ))}
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell>Haskell</TableCell>
+                <TableCell className='flex gap-2'>
+                  {['Stack'].map((p) => (
+                    <Badge
+                      className={`border ${getEcosystemBackgroundColor(p)}`}
+                    >
+                      {p}
+                    </Badge>
+                  ))}
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell>Java/Kotlin</TableCell>
+                <TableCell className='flex gap-2'>
+                  {['Gradle', 'GradleInspector', 'Maven'].map((p) => (
+                    <Badge
+                      className={`border ${getEcosystemBackgroundColor(p)}`}
+                    >
+                      {p}
+                    </Badge>
+                  ))}
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell>JavaScript/Node.js</TableCell>
+                <TableCell className='flex gap-2'>
+                  {['Bower', 'NPM', 'PNPM', 'Yarn', 'Yarn2'].map((p) => (
+                    <Badge
+                      className={`border ${getEcosystemBackgroundColor(p)}`}
+                    >
+                      {p}
+                    </Badge>
+                  ))}
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell>Kotlin/Scala</TableCell>
```

Comment:
Sorry, I seem to have missed this before, but SBT is the Scala Build Tool", so we should drop "Kotlin/" from here (which implies reordering due to sorting).